Add PR templates (#70)
AbsaOSS/hermes: .github/PULL_REQUEST_TEMPLATE/docs.md

---
name: Documentation PR
about: Create a PR for a GH Pages
title: 'ISSUE_NUMBER - Name'
labels: 'documentation'
assignees: ''

---

### Description

> - A small description if need
> - cause for this update
>     - link to an issue that

Reviewer: ```suggestion
- A small description if needed
- reason for this update
    - link to a related issue
```
